src/MoBi.Presentation/Presenter/SelectBuildingBlocksForExtendPresenter.cs

```diff
@@ -32,10 +33,12 @@ public SelectSpatialStructureAndMoleculesPresenter(ISelectSpatialStructureAndMol
          _mapper = mapper;
       }
 
-      public void SelectBuildingBlocksForExtend()
+      public void SelectBuildingBlocksForExtend(MoleculeBuildingBlock defaultMolecules, SpatialStructure defaultSpatialStructure)
       {
          setViewCaption();
          _dto = _mapper.MapFrom(_buildingBlockRepository.MoleculeBlockCollection, AllSpatialStructures.FirstOrDefault());
```

Comment:
here is used FirstOrDefault in th spatial structure but there is a deffaultSpatialStructure passed as parameter. Very confusing for me as I would expect the defaultSpatialStructure to be used on line 39

Reply:
Yeah, should only use FirstOrDefault if the passed in defaultSpSt is null that makes it much easier to understand